backend/entities/NoteTypes.cs

```diff
@@ -56,11 +56,11 @@ public enum NoteTypes
         /// <summary>
         /// SPP program cost notes.
         /// </summary>
-        SppCost = 12,
+        SplCost = 12,
```

Comment:
Your description still says SPP cost but the variable is SPL cost, not sure if this matters